.travis.yml

```diff
@@ -15,7 +17,9 @@ env:
     - ROS_PARALLEL_JOBS="-j8"
     - ROS_PARALLEL_TEST_JOBS="-j1"
     - CATKIN_PARALLEL_TEST_JOBS="-j1"
-    - USE_JENKINS=true
+    - USE_JENKINS=false
+    - USE_TRAVIS=true
```

Comment:
@wkentaro What does it mean if `USE_TRAVIS` and `USE_DOCKER` are both true?
When I created this option, I assumed either one is true.

Reply:
I'm not sure, so I set `UES_JENKINS=false` and `USE_TRAVIS=true`.